src/php/composer.json

```diff
@@ -1,20 +1,23 @@
 {
-  "name": "grpc/grpc-dev",
-  "description": "gRPC library for PHP - for Developement use only",
+  "name": "grpc/grpc",
+  "type": "library",
+  "description": "gRPC library for PHP",
+  "keywords": ["rpc"],
+  "homepage": "https://grpc.io",
   "license": "Apache-2.0",
-  "version": "1.7.0",
   "require": {
     "php": ">=5.5.0",
-    "google/protobuf": "^v3.3.0"
+    "google/protobuf": "^3.4.0"
   },
   "require-dev": {
     "google/auth": "v0.9"
   },
+  "suggest": {
+    "ext-protobuf": "For better performance, install the protobuf C extension."
+  },
   "autoload": {
     "psr-4": {
-      "Grpc\\": "lib/Grpc/",
```

Comment:
We need to keep this right? Relative to this directory `src/php`, the `Grpc\\` classes is under `lib/Grpc/` not `src/php/lib/Grpc`.

In fact, I think we don't really need to touch this file much at all.

Reply:
Woops, my bad. I've added it back

Reply:
I just realized the classes are actually under `src/lib`, which doesn't exist in this directory. it would be better if we could keep this path the same in `grpc/grpc-php`, i.e. `lib/Grpc`.